Makefile

```
@@ -311,12 +312,15 @@ REWRITER_FOLDER := rewriter
 REWRITER_SRC := $(REWRITER_FOLDER)/src
 COQPRIME_FOLDER := coqprime
 COQPRIME_SRC := $(COQPRIME_FOLDER)/src
-BEDROCK2_FOLDER := bedrock2/bedrock2
+BEDROCK2_FOLDER := rupicola/bedrock2/bedrock2
```

Might be a bit cleaner if you move the `RUBICOLA` variables up so you can do
```suggestion
BEDROCK2_FOLDER := $(RUBICOLA_FOLDER)/bedrock2/bedrock2
```